[CI] Makefile missing `run`

#### Current Behavior
```
➜  meshery-cilium git:(master) make run
make: *** No rule to make target `run'.  Stop.

```

#### Expected Behavior
`run` and other commands available in other Meshery adapters should be present.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/Makefile
+++ b/Makefile
@@ -1,24 +1,48 @@
-check:
+GOPATH = $(shell go env GOPATH)
+
+check: error
 	golangci-lint run
 
 check-clean-cache:
 	golangci-lint cache clean
 
 protoc-setup:
-	wget -P meshes https://raw.githubusercontent.com/layer5io/meshery/master/meshes/meshops.proto
+	cd meshes
+	wget https://raw.githubusercontent.com/layer5io/meshery/master/meshes/meshops.proto
 
 proto:
 	protoc -I meshes/ meshes/meshops.proto --go_out=plugins=grpc:./meshes/
 
-
-
-
-
-site:
-	$(jekyll) serve --drafts --livereload
-
-build:
-	$(jekyll) build --drafts
-
 docker:
-	docker run --name site -d --rm -p 4000:4000 -v `pwd`:"/srv/jekyll" jekyll/jekyll:4.0.0 bash -c "bundle install; jekyll serve --drafts --livereload"
+	docker build -t layer5/meshery-cilium .
+
+docker-run:
+	(docker rm -f meshery-cilium) || true
+	docker run --name meshery-cilium -d \
+	-p 10012:10012 \
+	-e DEBUG=true \
+	layer5/meshery-cilium
+
+run:
+	DEBUG=true go run main.go
+
+.PHONY: error
+error:
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . analyze -i ./helpers -o ./helpers
+
+.PHONY: local-check
+local-check: tidy
+local-check: golangci-lint
+
+.PHONY: tidy
+tidy:
+	@echo "Executing go mod tidy"
+	go mod tidy
+
+.PHONY: golangci-lint
+golangci-lint: $(GOLANGCILINT)
+	@echo
+	$(GOPATH)/bin/golangci-lint run
+
+$(GOLANGCILINT):
+	(cd /; GO111MODULE=on GOPROXY="direct" GOSUMDB=off go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.30.0)

